internal/wrappers/wrappers.go

```diff
@@ -54,46 +63,59 @@ type ISCSILib interface {
 
 // wrappers
 
+// OSExecWrapper contains implementation of LimitedOSExec interface
 type OSExecWrapper struct{}
 
+// CommandContext is a wrapper of exec.CommandContext
 func (w *OSExecWrapper) CommandContext(ctx context.Context, name string, arg ...string) LimitedOSExecCmd {
 	return exec.CommandContext(ctx, name, arg...)
 }
 
+// IOUTILWrapper contains implementation of LimitedIOUtil interface
 type IOUTILWrapper struct{}
 
+// ReadFile is a wrapper of ioutil.ReadFile
 func (io *IOUTILWrapper) ReadFile(filename string) ([]byte, error) {
 	return ioutil.ReadFile(filename) // #nosec G304
 }
 
+// FilepathWrapper contains implementation of LimitedFilePath interface
 type FilepathWrapper struct{}
 
+// Glob is a wrapper of filepath.Glob
 func (io *FilepathWrapper) Glob(pattern string) (matches []string, err error) {
 	return filepath.Glob(pattern)
 }
 
+// EvalSymlinks is a wrapper of filepath.EvalSymlinks
 func (io *FilepathWrapper) EvalSymlinks(path string) (string, error) {
 	return filepath.EvalSymlinks(path)
 }
 
+// OSWrapper contains implementation of LimitedOS interface
 type OSWrapper struct{}
 
+// OpenFile is a wrapper of os.OpenFile
 func (io *OSWrapper) OpenFile(name string, flag int, perm os.FileMode) (LimitedFile, error) {
-	return os.OpenFile(name, flag, perm)
+	return os.OpenFile(filepath.Clean(name), flag, perm) // #nosec G304
```

Comment:
After adding Filepath.clean, is nosec G304 requiredd?

Reply:
Even with filepath.Clean(), G304 error is reported by gosec. So added nosec here, as few other repos have excluded this check from gosec runs.

Reply:
this is correct fix,but still gosec is reporting issue